src/meshapi/views/forms.py

```diff
@@ -37,6 +39,42 @@ class JoinFormRequest:
     ncl: bool
 
 
+class JoinFormRequestSerializer(DataclassSerializer):
+    class Meta:
+        dataclass = JoinFormRequest
+
+
+form_err_response_schema = inline_serializer("ErrorResponse", fields={"detail": serializers.CharField()})
+
+
+@extend_schema_view(
+    post=extend_schema(
+        tags=["User Forms"],
+        auth=[],
+        summary="Register a new request for a potential mesh Install. "
+        "Used by the join form posted on the nycmesh.net website",
+        request=JoinFormRequestSerializer,
+        responses={
+            "201": OpenApiResponse(
+                inline_serializer(
+                    "JoinFormSuccessResponse",
+                    fields={
+                        "detail": serializers.CharField(),
+                        "building_id": serializers.IntegerField(),
+                        "member_id": serializers.IntegerField(),
+                        "install_number": serializers.IntegerField(),
+                        "member_exists": serializers.BooleanField(),
+                    },
+                ),
+                description="This install's building did not already have an install. One has been created and assigned.",
+            ),
+            "400": OpenApiResponse(
+                form_err_response_schema, description="Invalid request body JSON or missing required fields"
+            ),
+            "500": OpenApiResponse(form_err_response_schema, description="Unexpected internal error"),
+        },
+    ),
+)
```

Comment:
Is this purely for documentation purposes?

Reply:
Nah, let's keep it. If someone needs to work on the join form (For instance if they're working on the wizard onboarder idea) then they would need it.